Tests/FuzzilliTests/VariableMapTest.swift

```diff
@@ -59,30 +60,33 @@ class VariableMapTests: XCTestCase {
         XCTAssertEqual(m1, m2)
         
         // Add another 128 elements and compare with a new map built up in the opposite order
-        for i in 128..<256 {
+        for i in 127..<255 {
             let val = Bool.random()
             m2[v(i)] = val
         }
         
         var m3 = VariableMap<Bool>()
         XCTAssertNotEqual(m1, m3)
-        
-        for i in (0..<256).reversed() {
+
+        for i in (0..<255).reversed() {
```

Comment:
Same, `256`

Reply:
Changes reverted to the original implementation